src/info.rs

```diff
@@ -378,6 +391,34 @@ impl Info {
         output.lines().map(|x| x.to_string()).collect::<Vec<_>>()
     }
 
+    async fn get_number_of_tags_branches(dir: &str) -> (usize, usize) {
+        let tags = async {
+            let output = Command::new("git")
+                .args(vec!["-C", dir, "tag"])
+                .output()
+                .await
+                .expect("Failed to execute git.");
+
+            let tags = String::from_utf8_lossy(&output.stdout);
+
+            tags.lines().count()
+        };
+
+        let branches = async {
+            let output = Command::new("git")
+                .args(vec!["-C", dir, "branch"])
```

Comment:
Add `-r` flag to `git branch` otherwise you're only showing local branches. I think we're more interested with remote branches. And maybe try to filter out `origin/HEAD -> origin/master`